packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,646 @@
+                  "links": {
+                    "key": "https://docs.netlify.com/routing/redirects/redirect-options/#redirect-by-cookie-presence"
+                  }
+                }
+              },
+              "Country": {
+                "title": "Country",
+                "description": "The Country attribute accepts ISO 3166-1 alpha-2 country codes.",
+                "type": "array",
+                "items": {
+                  "type": "string",
+                  "pattern": "^[a-zA-Z]{2}$"
+                },
+                "x-taplo": {
+                  "links": {
+                    "key": "https://docs.netlify.com/routing/redirects/redirect-options/#redirect-by-country-or-language"
+                  }
+                }
+              },
+              "Language": {
+                "title": "Language",
+                "description": "The Language attribute accepts standard browser language identification codes and locale codes that combine language and country.",
+                "type": "array",
+                "items": {
+                  "type": "string",
+                  "pattern": "^[a-zA-Z]{2}(-[a-zA-Z]{2})?$"
+                },
+                "x-taplo": {
+                  "links": {
+                    "key": "https://docs.netlify.com/routing/redirects/redirect-options/#redirect-by-country-or-language"
+                  }
+                }
+              },
+              "Role": {
+                "title": "Role",
+                "description": "Role-based redirects let you restrict access to certain paths of your application to logged-in visitors with certain roles, as authorized by Netlify Identity or any authentication provider that supports JSON Web Tokens (JWT).",
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "x-taplo": {
+                  "links": {
+                    "key": "https://docs.netlify.com/routing/redirects/redirect-options/#redirect-by-role"
+                  }
+                }
+              }
+            }
+          },
+          "headers": {
+            "title": "Request headers",
+            "description": "Additional request headers to send in proxy redirects.",
+            "type": "object",
+            "additionalProperties": {
+              "type": "string"
+            },
+            "x-taplo": {
+              "docs": {
+                "main": "Additional request headers to send in [proxy redirects](https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects)."
+              },
+              "links": {
+                "key": "https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects"
+              }
+            }
+          },
+          "signed": {
+            "title": "Signed",
+            "description": "Name of an environment variable for signed proxy redirects.",
```

Just elaborating a bit:

```suggestion
            "description": "Name of the environment variable that contains the token to use for signed proxy redirects.",
```